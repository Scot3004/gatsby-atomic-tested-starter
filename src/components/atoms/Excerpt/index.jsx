import styled from 'styled-components';

export const Excerpt = styled.p`
  margin: 0;
`;

export default Excerpt;
