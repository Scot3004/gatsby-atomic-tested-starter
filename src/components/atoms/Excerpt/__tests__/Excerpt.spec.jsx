import React from 'react';
import 'jest-styled-components';
import renderer from 'react-test-renderer';
import Excerpt from '..';

test('it works', () => {
  const tree = renderer.create(<Excerpt />).toJSON();
  expect(tree).toMatchSnapshot();
});
