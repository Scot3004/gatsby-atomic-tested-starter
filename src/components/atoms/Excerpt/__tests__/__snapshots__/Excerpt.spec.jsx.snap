// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  margin: 0;
}

<p
  className="c0"
/>
`;
